test/cmdlineTests/wasm_to_wasm_function_returning_multiple_values/args

```diff
@@ -1 +1 @@
---yul --yul-dialect ewasm --machine ewasm
+--yul --yul-dialect ewasm --machine ewasm --bin --asm --ewasm --ewasm-ir --ir-optimized
```

Comment:
Currently you can only get output from either `--asm` or `--ewasm`, never both. `--asm` is generally not needed in any tests that use `ewasm` dialect and I'd leave only in cases where we actually want to verify that (e.g. in "debug info" tests). 
```suggestion
--yul --yul-dialect ewasm --machine ewasm --bin --ewasm --ewasm-ir --ir-optimized
```